subsys/bluetooth/controller/ll_sw/ctrl.c

```diff
@@ -1583,6 +1587,27 @@ static inline u32_t isr_rx_scan(u8_t devmatch_ok, u8_t devmatch_id,
 	return 1;
 }
 
+#if defined(CONFIG_BT_CONTROLLER_PHY)
+static inline void isr_rx_conn_phy_tx_time_set(void)
+{
+	struct connection *conn = _radio.conn_curr;
+	u8_t phys = conn->llcp_phy.tx | conn->phy_tx;
+
+	/* select the probable PHY with longest Tx time, which will be
+	 * restricted to fit current connEffectiveMaxTxTime.
+	 */
+	if (phys & BIT(2)) {
```

Comment:
would it be worth using a look-up table here instead of the succession of if() statements?